docs/field-descriptions.yaml

```diff
@@ -77,6 +77,7 @@ execution_datetime: Start date and time of assay, typically a date-time stamped
   generated by the acquisition instrument. YYYY-MM-DD hh:mm, where YYYY is the year,
   MM is the month with leading 0s, and DD is the day with leading 0s, hh is the hour
   with leading zeros, mm are the minutes with leading zeros.
+expected_cell_count: Parameter for downstream pipeline; How many cells are expected?
```

Comment:
Might be better as "How many cells are expected? This may be used in downstream pipelines to guide selection of cell barcodes or segmentation parameters."

I.e., we'll use this in the Salmon s*RNA-seq pipeline very soon, but not yet scATAC-seq or any imaging modalities, but that may change in the future.

Reply:
fixed. Thanks!